examples/hello_redisearch.rs

```diff
@@ -21,7 +21,7 @@ fn hello_redisearch(_: &Context, args: Vec<String>) -> RedisResult {
     let score = 1.0;
 
     let index = Index::create(index_name);
-    index.create_field(field_name);
+    index.create_field(field_name, 1.0, None, false);
```

Comment:
This argument list is very C-like in that the arguments are not self-documenting: What does `None` mean? What is `false`? To answer this the user needs to look at the implementation.

There are two ways to solve this:
1. Use a `struct` to pass the arguments and name the fields explicitly.
2. Use the builder pattern, as in https://doc.rust-lang.org/stable/std/fs/struct.OpenOptions.html.